components/properties/src/names.rs

```diff
@@ -129,58 +125,6 @@ impl<T> PropertyParser<T> {
 }
 
 impl<T: TrieValue> PropertyParserBorrowed<'_, T> {
-    /// Get the property value as a u16, doing a strict search looking for
-    /// names that match exactly
-    ///
-    /// # Example
-    ///
-    /// ```
-    /// use icu::properties::PropertyParser;
-    /// use icu::properties::props::GeneralCategory;
-    ///
-    /// let lookup = PropertyParser::<GeneralCategory>::new();
-    /// assert_eq!(
-    ///     lookup.get_strict_u16("Lu"),
-    ///     Some(GeneralCategory::UppercaseLetter as u16)
-    /// );
-    /// assert_eq!(
-    ///     lookup.get_strict_u16("Uppercase_Letter"),
-    ///     Some(GeneralCategory::UppercaseLetter as u16)
-    /// );
-    /// // does not do loose matching
-    /// assert_eq!(lookup.get_strict_u16("UppercaseLetter"), None);
-    /// ```
-    #[inline]
-    pub fn get_strict_u16(&self, name: &str) -> Option<u16> {
-        get_strict_u16(self.map, name)
-    }
```

Comment:
Issue: We need `get_strict` for spec compliance; I think you can implement it using ZeroAsciiIgnoreCaseTrie

Reply:
Does the caae-insensitive trie support strict lookup? Are values guaranteed to never share differently-cased prefixes, which the case-insensitive trie can't store?

Reply:
It does. I think right now you need to use the cursor API. The `.step()` function does a case-insensitive lookup but returns the actual u8; for a strict lookup, you can then to check if the input u8 matches the actual u8. We should probably add a dedicated `.get_strict` function.

Reply:
As far as whether the values could conflict with each other: I don't know, but you can check and see if they work.

Reply:
https://github.com/unicode-org/icu4x/pull/5585

Reply:
The values conflict, so we cannot use `ZeroAsciiIgnoreCaseTrie`.